Makefile

```diff
@@ -15,13 +17,16 @@ TEST_COV = --coverage # --coverage is a synonym for-fprofile-arcs, -ftest-covera
 # TODO: Split out dependency handling so that we can avoid rebuilding
 .PHONY: $(TARGET) $(TESTS) all test integration_test clean
 
+# Backslashes at the end of lines in an addprefix 'block' need to be immediately
+# after the file name, since arguments to addprefix are separated by a single
+# space and Make converts '\' into a single space
 TARGET = lightning
 TESTS = server_config_test
-SRC = $(PARSER_PATH)config_parser.cc lightning_main.cc \
-	lightning_server.cc server_config.cc mime_types.cc \
-	request_handlers.cc request_router.cc request.cc response.cc \
-	server_stats.cc $(MD_PATH)markdown.cpp \
-	$(MD_PATH)markdown-tokens.cpp
+SRC = $(PARSER_PATH)config_parser.cc \
+			$(addprefix $(SRC_PATH), lightning_main.cc lightning_server.cc\
```

Comment:
That is a neat little function, `addprefix`. For reference, here are Make's [other utility functions ](http://www.chemie.fu-berlin.de/chemnet/use/info/make/make_8.html)for text-transformation.